lib/ui/window.dart

```diff
@@ -51,18 +51,22 @@ part of dart.ui;
 /// the [viewPadding] anyway, so there is no need to account for
 /// that in the [padding], which is always safe to use for such
 /// calculations.
-///
-/// See also:
-///
-///  * [FlutterWindow], a special case of a [FlutterView] that is represented on
-///    the platform as a separate window which can host other [FlutterView]s.
-abstract class FlutterView {
+class FlutterView {
+  FlutterView._(this._viewId, this.platformDispatcher);
+
+  /// The opaque ID for this view.
+  final Object _viewId;
```

Comment:
move the private field below the getter. Our style is to have the following order: getter, private field, setter. The setter may be optional, though.

Reply:
Sure thing. 

Is that documented somewhere?

Reply:
I actually couldn't find this in the style guide, but it is something we try to follow. We should add it.